main/src/vtr_path_planning/src/mpc/mpc_path_planner.cpp

```diff
@@ -263,31 +366,116 @@ struct mpc_result SolveMPC(Eigen::Matrix<double, 2, 1> previous_vel, lgmath::se3
       mpc_poses.push_back(pose_state_vars[i]->value().inverse());
     }
 
-    
     // Return the resulting structure
     return {applied_vel, mpc_poses};
     }
 }
 
 
+/*
+// Measurement generation overhaul, not quite ready to deploy yet
```

Comment:
Does that mean it is coming?

Reply:
ooop forgot to remove this - ended up scrapping this early prototype function and just integrated some of the ideas into the existing function - will remove comments